include/votca/tools/calculator.h

```diff
@@ -77,20 +77,40 @@ class Calculator {
    * @param output stream
    */
   void DisplayOptions(std::ostream &out);
+
   /**
-   * \brief Updates options with default options stored in VOTCASHARE
+   * \brief Loads default options stored in VOTCASHARE
+   */
+  Property LoadDefaults(const std::string package = "tools");
+
+  /**
+   * \brief Updates user options with default options stored in VOTCASHARE
    *
    * If a value is not given or tag is not present and at the same time
    * a default value exists in the corresponding XML file in VOTCASHARE
    * a tag is created and/or a default value is assigned to it
    */
-  void UpdateWithDefaults(Property &options, std::string package = "tools");
+  void UpdateWithUserOptions(Property &default_options,
```

Comment:
I think more of the functions are `const`. Could you please check?

Reply:
I think that Initialize was the last missing `const`